DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -42,6 +43,7 @@ public string RegisterNewUserAndDelegateAccount(DelegateRegistrationModel delega
                 delegateRegistrationModel.Active,
                 PasswordHash = "temp",
                 ProfessionalRegistrationNumber = (string?)null,
+                DetailsLastChecked = DateTime.UtcNow,

```

Comment:
A bit of nitpicking, but this `DateTime.UtcNow` and the one you use to update `DelegateAccounts` table further down in this method will be slightly different. I would expect them to have the same value because we update them together every time.